### PR TITLE
Cluster Resource Fixes

### PR DESCRIPTION
**Pull request by Krunal-Thakkar**
# Description
Cluster Resource Fixes

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: powerflex/provider/powerflex.env [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
+POWERFLEX_SDS_IP_8=10.10.10.12
+POWERFLEX_SDS_IP_9=10.10.10.13
+POWERFLEX_SDS_IP_10=10.10.10.14
+POWERFLEX_SDS_IP_11=10.247.101.244
+POWERFLEX_SDC_IP1=10.247.66.67
+POWERFLEX_SDC_VOLUMES_MAPPING_NAME=tf-unknown-test-donot-delete
+POWERFLEX_PROTECTION_DOMAIN_ID=202a046600000000
+POWERFLEX_STORAGE_POOL_NAME=terraform-pool
+POWERFLEX_PRIMARY_MDM_IP=10.247.66.67
+POWERFLEX_SECONDARY_MDM_IP=10.247.100.214
```
Comment (AnikaAgiwal2711) [inline powerflex/provider/powerflex.env:21]: Please remove these ips

Comment (Krunal-Thakkar) [reply powerflex/provider/powerflex.env]: Updated.

**Comment by shenda1** [comment]
Looks good to me

**Comment by Krunal-Thakkar** [comment]
run e2e test

**Comment by Krunal-Thakkar** [comment]
run e2e test

**Comment by Krunal-Thakkar** [comment]
run e2e test

**Review by shenda1 — APPROVED** [review]

**Review by AnikaAgiwal2711 — APPROVED** [review]

**Pull request merged**